cola/widgets/selectcommits.py

```diff
@@ -56,11 +56,18 @@ def __init__(self, context, model, parent=None, title=None, multiselect=True):
 
         self.commit_text = DiffTextEdit(context, self, whitespace=False)
 
-        self.label = QtWidgets.QLabel()
-        self.label.setText(N_('Revision Expression:'))
+        self.revsion_label = QtWidgets.QLabel()
```

Comment:
Typo?

Reply:
No had to create a label for Search bar.
So renamed label -> revision_label
and created a label for search named search_label.

Reply:
Link to the issue https://github.com/git-cola/git-cola/issues/795

Reply:
It says `revsion`, not `revision` :slightly_smiling_face:

Reply:
Ahh will fix that.

Reply:
Fixed the typo.